src/commons/eventsourcing/store.rs

```diff
@@ -506,6 +507,9 @@ where
 
                     cache.insert(handle.clone(), Arc::new(agg.clone()));
 
+                    // Save the info so that the aggregate can be loaded properly by the listeners.
+                    self.save_info(&handle, &info)?;
```

Comment:
With the changes in ca-parent sync scheduling it could happen that the event listener saw an update *before* the info file was committed - and it would then try to use an older version of the CA.

I think that this existed before as well except that it was very rarely hit - because the tasks were added and only checked every couple of seconds. So there was normally a delay then.